cobigen-cli/cli/src/main/java/com/devonfw/cobigen/cli/commands/GenerateCommand.java

```diff
@@ -124,19 +124,23 @@ public Integer call() throws Exception {
                 if (inputFiles.size() > 1) {
                     finalTemplates = toTemplateTo(preprocess(cg, TemplateTo.class));
                 }
+
+                ClassLoader inputClassLoader = getClass().getClassLoader();
```

Comment:
Have you tested this with the CobiGen CLI fat jar? What I mean is:

1. Run `mvn clean install` on the CLI. That will create a CobiGen.jar inside target folder (our users will install that).
2. Execute `cobigen.bat` or either open a console and execute `java -javaagent:"%~dp0\class-loader-agent.jar" -jar "%~dp0\cobigen.jar" %*` where in `%*` you need to put the correct arguments like `generate EmployeeEntity.java`

Reply:
Tested and seems to work.

Reply:
Great, thanks!